drivers/storage/vbox/storage/vbox_storage.go

```diff
@@ -674,7 +675,7 @@ func (d *driver) attachVolume(
 		return err
 	}
 
-	if err := m.Refresh(); err != nil {
+	if err = m.Refresh(); err != nil {
```

Comment:
Yes it was the shadow warning. It's still not clear to me what the right thing to do there is, as you make a good point about scoping.